plugins/modules/inventory_intent.py

```diff
@@ -2460,6 +2517,20 @@ def update_interface_detail_of_device(self, device_to_update):
                 # Now we call update interface details api with required parameter
                 try:
                     interface_params = self.config[0].get('update_interface_details')
+                    clear_mac_address_table = interface_params.get("clear_mac_address_table", False)
+
+                    if clear_mac_address_table:
+                        response = self.get_device_response(device_ip)
+
+                        if response.get('role').upper() != "ACCESS":
+                            self.msg = "Clearing MAC Address action is only supported on device with ACCESS role"
```

Comment:
self.msg = "The action to clear the MAC Address table is only supported for devices with the ACCESS role."

Reply:
updated.